tests/stubtest_allowlists/py310.txt

```diff
@@ -135,3 +135,84 @@ ssl._create_unverified_context
 typing.SupportsRound.__round__
```

Comment:
While we're here, could we move this to the correct section? I added it in one of my first PRs to typeshed (#6242), and put it in the wrong section.